strict-checked-vars/strict-checked-vars.cabal

```
@@ -90,3 +91,9 @@ test-suite test
     -Wall -Wcompat -Wincomplete-uni-patterns
     -Wincomplete-record-updates -Wpartial-fields -Widentities
     -Wunused-packages -fno-ignore-asserts
+
+  if flag(checkmvarinvariants)
```

I don't know if it is too late for this, but I think cabal flags allow dashes? It could be worth adding them for improved readability. (same for `check-tvar-invariants`)
```suggestion
  if flag(check-mvar-invariants)
```

---

Good point, but I'd rather not change the flag now and break builds silently, since `cabal` won't complain if a flag doesn't exist

---

It is a pain in the neck that cabal doesn't warn about non-existent flags, and honestly I don't know why this is the case. But in this state, I'd rather not modify the flag name too.

---

I think retaining the old flag would allow for non-silent transition:
```
  if flag(checktvarinvariants)
    -- use `check-tvar-invariants` instead
    buildable: false
```

I only made this suggestion because difference between `m` vs `t` in these two flags is not very pronounced: checktvarinvariants vs checkmvarinvariants